addons/marketing_automation/models/mass_mailing.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class MassMailing(models.Model):
+
+    _inherit = 'mail.mass_mailing'
+
+    is_automation = fields.Boolean(default=False)
+    automation_model_name = fields.Char()
+
+    @api.depends('mailing_model')
```

Comment:
That doesn't seem correct. `mailing_model` is used nowhere in the compute, but `is_automation` and `automation_model_name` are...